src/622/modules/fios-sequenciados/index.js

```diff
@@ -0,0 +1,50 @@
+// Quais aparicoes devem ser cortadas
+const rules = {
+    "red": {
+        1: ["C"], 2: ["B"], 3: ["A"],
+        4: ["A", "C"], 5: ["B"], 6: ["A", "C"],
+        7: ["A", "B", "C"], 8: ["A", "B"], 9: ["B"],
+    },
+    "blue": {
+        1: ["B"], 2: ["A","C"], 3: ["B"],
+        4: ["A"], 5: ["B"], 6: ["B", "C"],
+        7: ["C"], 8: ["A", "C"], 9: ["A"],
+    },
+    "black": {
+        1: ["A", "B", "C"], 2: ["A","C"], 3: ["B"],
+        4: ["A", "C"], 5: ["B"], 6: ["B", "C"],
+        7: ["A", "B"], 8: ["C"], 9: ["C"],
+    }
+}
+
+/**
+   * Função que resolve o módulo fios sequenciados
+   * @param {Object} lastStep - Objeto que traz a informação das interações anteriores
+   * @param {Object} currentStep - Objeto que traz a informação da interação atual
+   * @returns {string[]} Array com o objeto newLastStep na posição 0, que é o resultado do merge to lastStep com o current Step e três strings podendo ser true, false e null
+   */
+   
+const seqwires = (lastStep, currentStep) => {
+    const newLastStep = {...lastStep, ...currentStep}
```

Comment:
Não tem nenhuma validação para caso o currentstep não seja enviado ou não seja um objeto como o esperado. Se for uma string, bill ou undefined ja vai quebrar, melhor configurar um default na assinatura da função e validar se é um objeto antes de usar o spread operator